### PR TITLE
[misc] Create controllers for features and disorders

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

File: components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/DisordersController.java [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        } catch (Exception e) {
+            this.logger.error("Failed to access patient data for [{}]: {}", patient.getDocument(), e.getMessage());
+        }
+        return null;
+    }
+
+    @Override
+    public void writeJSON(Patient patient, JSONObject json, Collection<String> selectedFieldNames)
+    {
+        if (selectedFieldNames != null && !selectedFieldNames.contains(DISORDER_PROPERTIES)) {
```
Comment (sdumitriu) [inline components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/DisordersController.java]: I don't think this works, should be `contains(DISORDER_PROPERTIES_OMIMID)`.

File: components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/DisordersController.java [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        } catch (Exception e) {
+            this.logger.error("Could not load disorders from JSON: [{}]", e.getMessage(), e);
+            return null;
+        }
+    }
+
+    @Override
+    public void save(Patient patient, DocumentModelBridge doc)
+    {
+        PatientData<Disorder> features = patient.getData(this.getName());
```
Comment (sdumitriu) [inline components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/DisordersController.java]: Should be called `disorders`, not `features`.

File: components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/DisordersController.java (hunk 1 above)
Comment (veronikaslc) [inline components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/DisordersController.java]: Then why we have it like that here https://github.com/phenotips/phenotips/blob/master/components/patient-data/impl/src/main/java/org/phenotips/data/internal/PhenoTipsPatient.java#L355 ?

**Comment by veronikaslc** [comment]
@sdumitriu comments addressed

**Pull request merged**